Formula/opencv.rb

```diff
@@ -57,7 +57,9 @@ def install
       -DBUILD_TESTS=OFF
       -DBUILD_TIFF=OFF
       -DBUILD_ZLIB=OFF
+      -DBUILD_opencv_hdf=OFF
       -DBUILD_opencv_java=OFF
+      -DBUILD_opencv_text=OFF
```

Comment:
Hey @commitay!
I believe this line is a bit problematic here. Just spent the last 30 minutes trying to figure out why some of my projects which depend on opencv suddenly stopped working after a `brew update & brew upgrade` with the following error: `Library not loaded: /usr/local/opt/opencv/lib/libopencv_text.3.4.dylib`

Reply:
Did you try rebuilding them? Linkage to `libopencv_text.3.4.dylib` should not be needed following a rebuild of the dependent project.

Reply:
Ah, yes! You are right, a rebuild will do the trick.